packages/contracts/contracts/voting/majority/IMajorityVoting.sol

```diff
@@ -49,22 +51,22 @@ interface IMajorityVoting {
     event VoteExecuted(uint256 indexed voteId, bytes[] execResults);
 
     /// @notice Emitted when the vote configuration is updated.
-    /// @param participationRequiredPct The required participation in percent.
-    /// @param supportRequiredPct The required support in percent.
+    /// @param relativeSupportThresholdPct The relative support threshold in percent.
+    /// @param totalSupportThresholdPct The total support threshold in percent.
     /// @param minDuration The minimal duration of a vote.
     event ConfigUpdated(
-        uint64 participationRequiredPct,
-        uint64 supportRequiredPct,
+        uint64 totalSupportThresholdPct,
+        uint64 relativeSupportThresholdPct,
         uint64 minDuration
     );
 
     /// @notice Sets the vote configuration.
-    /// @param _participationRequiredPct The required participation in percent.
-    /// @param _supportRequiredPct The required support in percent.
+    /// @param _totalSupportThresholdPct The total support threshold in percent.
+    /// @param _relativeSupportThresholdPct The relative support threshold in percent.
     /// @param _minDuration The minimal duration of a vote.
     function setConfiguration(
-        uint64 _participationRequiredPct,
-        uint64 _supportRequiredPct,
+        uint64 _relativeSupportThresholdPct,
+        uint64 _totalSupportThresholdPct,
         uint64 _minDuration
```

Comment:
These 2 names feel very confusing. Total vs relative in terms of what?

Also, we cannot represent a percent using a uint256. We can express ratios based on an upper bound, defined per-plugin

Reply:
This will be worked on in task https://aragonassociation.atlassian.net/browse/APP-1250